### ISSUE TITLE
Fetching 'abe' submodule failed

### ISSUE THREAD

**Issue by ghost** [ali1234/rpi-toolchain#1]
I'm trying to build your repo to cross-compile a decent toolchain for Rpi3B+. I cloned recursively the repository but here the last part of the output:

```
Cloning into '/home/femtotech/opt/rpi-toolchain/abe'...
remote: Enumerating objects: 141, done.        
remote: Counting objects: 100% (141/141), done.        
remote: Compressing objects: 100% (116/116), done.        
remote: Total 141 (delta 20), reused 24 (delta 1)        
Ricezione degli oggetti: 100% (141/141), 191.68 KiB | 0 bytes/s, done.
Risoluzione dei delta: 100% (20/20), done.
Fetched in submodule path 'abe', but it did not contain bb166f788ba4180755c8cdbc9c63078446307b85. Direct fetching of that commit failed.
```

the 'abe' folder is empty.

**Comment by ali1234** [comment]
This again. You need to do a full (not shallow) clone. I don't know why git even has this option to make shallow clones since it basically never works.

**Comment by ali1234** [comment]
Fixed in b57db8d
